README.rst

```diff
@@ -37,9 +37,20 @@ Just import the decorator, use, and return a JSON-serializable object::
         }
 
 
-`Class-based views`_ (CBVs) can either use Django's
-``@method_decorator`` or can wrap the output of ``.as_view()``::
+`Class-based views`_ (CBVs) can inherit from JsonView, use Django's
+``@method_decorator`` or wrap the output of ``.as_view()``::
 
+    # inherit from JsonView
+    from jsonview.views import JsonView
+
+
+    class MyView(JsonView):
+        def get_context_data(self, **kwargs):
```

Comment:
Is this a common pattern in CBVs? How would I write a `POST` handler?

Reply:
To be honest, it's been so long since I made this fork, I don't remember my thinking at the time. I've actually been using my fork in my own projects for several years now. If I had to guess, I'd say I used get_context_data because this is the first extension of class-based views taught in the Django docs, and often, the only method you need to override.

Reply:
I have not tried POST. I could try extending the class to handle POST requests.

Reply:
I _think_, since you’re wrapping `dispatch`, that all methods should be fine. I’m not sure why `get()` delegates all its work to `get_context_data`—and if users should/would just override `get()` instead—instead of doing it directly. And if there is a good reason, why is it only `get`?

Reply:
I would guess I was working from the TemplateView code:
```
    def get(self, request, *args, **kwargs):
        context = self.get_context_data(**kwargs)
        return self.render_to_response(context)
```
The decorator handles the rendering part, so I just made the view return context. Originally, I was decorating the entire class rather than overriding dispatch, but that only works in Django 1.9+ so I changed it in the pull request. I think for post the user would have to override it themselves, as there is no logical base behaviour I can think of.